Remotion/ObjectBinding/Web/UI/Controls/BocDropDownMenu.cs

```diff
@@ -202,14 +202,15 @@ private void PreRenderDropDownMenu()
     }
   }
 
-  protected virtual string GetTitleText (IBusinessObject businessObject)
+  protected virtual WebString GetTitleText (IBusinessObject businessObject)
   {
     ArgumentUtility.CheckNotNull ("businessObject", businessObject);
+
+    var titleText = businessObject is IBusinessObjectWithIdentity
+        ? ((IBusinessObjectWithIdentity) businessObject).GetAccessibleDisplayName()
+        : businessObject.ToString();
```

Comment:
```suggestion
    var titleText = businessObject is IBusinessObjectWithIdentity
        ? ((IBusinessObjectWithIdentity) businessObject).GetAccessibleDisplayName()
        : businessObject.ToString();
```
```suggestion
    var titleText = businessObject switch
    {
        IBusinessObjectWithIdentity businessObjectWithIdentity => businessObjectWithIdentity.GetAccessibleDisplayName(),
        _ => businessObject.ToString()
    };
```

Do you think we should do a switch expression instead. The semantic of the IF is a type switch so it feels more correct to do pattern matching instead of an if/ternary. Thoughts?

Reply:
In this case I think doing a switch expression does not add much because we only have two branches, but I changed the ternary operator to use pattern matching, removing the need for casting